src/pykeen/models/nbase.py

```diff
@@ -180,6 +184,7 @@ def score_hrt(self, hrt_batch: torch.LongTensor) -> torch.FloatTensor:
             h_indices=hrt_batch[:, 0],
             r_indices=hrt_batch[:, 1],
             t_indices=hrt_batch[:, 2],
+            # TODO can slices be used here?
```

Comment:
Consider updating this comment (but not necessarily providing an implementation) in this PR

Reply:
slicing cannot be used here: the indices for `score_hrt` only have a batch dimension, and slicing along this dimension is already considered by sub-batching.